engine/src/main/java/io/zeebe/engine/processor/StreamProcessor.java

```diff
@@ -164,6 +159,7 @@ protected void onActorCloseRequested() {
 
   @Override
   protected void handleFailure(final Exception failure) {
+    LOG.error("Actor {} failed in phase {}.", actorName, actor.getLifecyclePhase(), failure);
```

Comment:
shouldnt this log be a level higher in the actor class?

Reply:
We have a log in the actor class. But it says, "actor continue with the next job." But here we are not continuing with next jobs, but want to log that it failed.

Reply:
I thought we don't want to continue anymore :thinking:

Reply:
Default behavior is we continue, unless the actor decides to fail by overriding 'handleFailure()`

Reply:
https://github.com/zeebe-io/zeebe/pull/4009#discussion_r390290294

Reply:
sorry